app/models/concerns/backup_code_encrypted_attribute_overrides.rb

```diff
@@ -0,0 +1,28 @@
+module BackupCodeEncryptedAttributeOverrides
+  extend ActiveSupport::Concern
+
+  class_methods do
+    # :reek:UtilityFunction
+    def find_with_code(code)
+      BackupCodeConfiguration.find_with_code(code)
```

Comment:
This needs coverage.

Reply:
Ok.  Will add that.  I was about to reassign it to you anyway. :)

Reply:
Done

Reply:
It's still saying that it's not covered.

Reply:
There were two implementations of that method.  That one was unused.  I removed it.